Tests/TuistGeneratorTests/ProjectMappers/GenerateModuleMapProjectMapperTests.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistGraphTesting
+import TuistSupport
+import XCTest
+@testable import TuistCoreTesting
+@testable import TuistGenerator
+@testable import TuistSupportTesting
+
+public final class GenerateModuleMapProjectMapperTests: TuistUnitTestCase {
+    var subject: GenerateModuleMapProjectMapper!
+
+    override public func setUp() {
+        super.setUp()
+        subject = GenerateModuleMapProjectMapper(
+            derivedDirectoryName: Constants.DerivedDirectory.name,
+            moduleMapsDirectoryName: Constants.DerivedDirectory.moduleMaps
+        )
+    }
+
+    override public func tearDown() {
+        super.tearDown()
```

Comment:
should be the last - I should really look into if this can be caught by a linter 😄